Tests/Marketplace/marketplace_services.py

```diff
@@ -1582,7 +1583,7 @@ def prepare_release_notes(self, index_folder_path, build_number, modified_rn_fil
                                 logging.debug(f"{all_relevant_pr_nums_for_unified=}")
                                 updated_entry = self._get_updated_changelog_entry(
                                     changelog, version, release_notes=modified_release_notes_lines,
-                                    pull_request_numbers=all_relevant_pr_nums_for_unified)
+                                    pull_request_numbers=all_relevant_pr_nums_for_unified, marketplace=marketplace)
```

Comment:
Can you remind me why we had this change?

Reply:
In case someone is updating an old RN file then we will want to filter the updated RN entry (which should have the marketplace to which the upload is done)